Assignment2/question6.js

```diff
@@ -0,0 +1,8 @@
+//function expression that takes in a number and returns true if it's even and false if it's odd
+
+let isEven = function (number) {
```

Comment:
use const if value is not being updated,
Suggestion is to use ESLint to get such suggestions.
Read about ES Lint and its usages.